### PR TITLE
Some things

### PR DESCRIPTION
**Pull request by MemeHoovy**
Deleted useless files, including some leftover stuff from the Lundum dare.
Added some things to Hscript (Untested, due to lack of time, it's like almost 3 AM, and I'm getting tired.)
And also other things I didn't document.

I'll make this a draft if I need to btw.

### CODE REVIEW

File: source/ChartParser.hx [hunk 1]
```diff
@@ -1,83 +0,0 @@
-package;
```
Comment (MemeHoovy) [inline source/ChartParser.hx:1]: This is a leftover from the lundum dare, and as such, it's unneeded.

**Comment by MemeHoovy** [comment]
I also allowed edits so in-case you find a solution for Hscript

**Comment by UmbratheUmbreon** [comment]
Not quite sure exactly why HScript isnt working still, though i suppose it could be an error with calling functions maybe.
If not the only other thing i could think of is the instance being weird

**Comment by UmbratheUmbreon** [comment]
Oh actually now that i think about it, it probably cant find mods folder hscripts and thats why it isnt working HMMMMM
Truly the greatest oversight of all time

**Comment by UmbratheUmbreon** [comment]
This pr has been implemented into the dev branch so im gonna close it now

**Pull request closed without merging**